### DISCUSSION TITLE
mining reward moved from one address and now gone

### DISCUSSION THREAD

**Discussion by bitgiguere** [Chia-Network/chia-blockchain#8865] [Support]
I had a mining reward of .25 and then 1.75 Chia. And after using the official Chia faucet. My 1.75 Chia moved to 2 addresses that both seem to belong to my private key. Then on the third move it sent 2 chia to another address that does not show up in my wallet. In the wallet I can only see incoming transactions. So I can see the amount coming into my wallet 3 times. No outgoing at all. The only amount that moved was the reward amount. The move started from the address below:
xch16n4kqmrtrk3hywhjcz3vwd9du3lv3x2fhx0r9pqjsedcm4k0ga4s8lq4ad

I have redownloaded the blockchain and still only see the same transactions. The only way to follow the Chia is through the explorer. Is this money actually just gone? why would the 1.75 move twice and then finally the full 2 move to where my wallet cannot see it?

Thanks for any help

**Comment by BrandtH22** [comment]
@bitgiguere , unfortunately it appears someone has gained access to your keyset and/or device and stolen your funds. Since chia is a truly decentralized network there is no way to reclaim the funds or revert the transaction(s). One thing we recommend is ensuring that you and your farm are following security best practices to reduce malicious actors ability to gain access to your keys and/or device, many of these best practices are outlined here: https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/

Since we unfortunately cannot do more to assist with the stolen funds we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
